### DISCUSSION TITLE
How to implement skipping method execution in plugin?

### DISCUSSION THREAD

**Discussion by JihanLin** [apache/skywalking#10781] [Q&A] [answered]
Hello,
 I want to implement skipping a method before it is executed.please, how to do it in plugin? in InstanceMethodsAroundInterceptor.beforeMethod this is not returned operations.

**Comment by wu-sheng** [comment, marked as answer]
Use `MethodInterceptResult result` in the parameter list.

**Reply by JihanLin** [reply]
Thank u
